app/src/pages/organization/projectTeamPage/projectTeamListTable/projectTeamListTable.jsx

```diff
@@ -0,0 +1,175 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { activeProjectKeySelector, userRolesSelector } from 'controllers/user';
+import { AbsRelTime } from 'components/main/absRelTime';
+import { MeatballMenuIcon, Popover, Table } from '@reportportal/ui-kit';
+import { UserAvatar } from 'pages/inside/common/userAvatar';
+import { urlOrganizationAndProjectSelector } from 'controllers/pages';
+import { SORTING_ASC, withSortingURL } from 'controllers/sorting';
+import { DEFAULT_SORT_COLUMN } from 'controllers/administrate/allUsers';
```

Comment:
I suppose the `administrate/allUsers` page and controller will be reorganized soon, so let's create own `DEFAULT_SORT_COLUMN` in members controller.

Reply:
Done.